docs/user-guide/certificate-configuration-scenarios.md

```diff
@@ -290,6 +289,13 @@ this field is not defined, the `zwe init` command uses the value `zowe.externalD
      - "zOSMFCA"
 ```
 
+If you would like to use this example in your Zowe configuration YAML file, you just need to replace four fields with your own ones.
+
+1. Replace `ZoweKeyringZOSMF` with the your own key ring name.
+2. Replace `IZUSVR` with your user name.
+3. Replace `DefaultzOSMFCert.IZUDFLT` with your own label name.
```

Comment:
The label is the one for the existing certificate, so maybe that should be described differently just in case people think it's free form, e.g.
3. Replace 'DefaultzOSMFCert.IZUDFLT' with the label of the existing certificate you're connecting to (which will be owned by the user ID above).